frontend/containers/discover-map/component.tsx

```diff
@@ -148,6 +154,24 @@ export const DiscoverMap: FC<DiscoverMapProps> = ({ onSelectProjectPin }) => {
           <MapHelp />
         </div>
 
+        <div className="absolute bottom-2 left-4">
+          {displayResourceWatchCredits && (
+            <span className="text-xs text-gray-800 drop-shadow-[1px_1px_0px_rgb(198,198,198)]">
+              <FormattedMessage
+                defaultMessage="Powered by <a>Resource Watch</a>"
+                id="vrCHpK"
+                values={{
+                  a: (chunk: string) => (
+                    <a className="hover:underline" href="https://resourcewatch.org/">
+                      {chunk}
+                    </a>
```

Comment:
We can make some small improvements to the link:
```suggestion
                    <a
                      className="hover:underline"
                      href="https://resourcewatch.org/"
                      target="_blank"
                      rel="noopener noreferrer"
                    >
                      {chunk}
                    </a>
```